javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,46 @@
+
+//Flight booking fullname function
+
+function getFullName(firstName, surname, useFormalName = true, gender = "") {
+    if (useFormalName) {
+        if (gender === "male") {
+            return "lord " + firstName + surname;
+        } else if (gender === "female") {
+            return "lady " + firstName + surname;
+        }
+    }
+            return firstName + surname;
+        }
```

Comment:
The solution is very creative, and it is a very good usage of conditions.
Though it is hard to read with the current formatting. 
Using "else" is very useful for ensuring a clean and readable code
`else {
          return firstName + surname;
         }`